provider/idps/attribute_validator.go

```diff
@@ -0,0 +1,24 @@
+package idps
+
+import (
+	"context"
+	"github.com/hashicorp/terraform-plugin-framework/tfsdk"
+)
+
+type attributeValidator struct {
```

Comment:
Maybe this struct can be shared with other resources?  
If no change the name to include IDP